static/src/javascripts/projects/common/modules/tracking/google-analytics.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+import mediator from 'lib/mediator';
+import {onConsentChange, getConsentFor} from '@guardian/consent-management-platform';
+
+export const init: () => void = () => {
+  onConsentChange(state => {
```

Comment:
Can there be a situation where consent is already granted at this point and you'll miss the change event and end up not loading GA? Do you need to check for consent immediately *and* wait for change?

Reply:
I'm not sure, it hasn't been a problem testing locally, but that doesn't mean this isn't a problem! Let me find 15mins to talk this through tomorrow.

Reply:
This shouldn't be an issue!

`onConsentChange` will still run if consent has been initialised previously:

> If the consent state has already been acquired when onConsentChange is called, the callback will be invoked immediately.
 
https://github.com/guardian/consent-management-platform#onconsentchangecallback

cc. @sndrs

Reply:
Ah great, thanks - I shoulda read the docs!

Reply:
To be fair, it’s a little bit of a misnomer.